### PR TITLE
build: cd firebase hosting

### PR DESCRIPTION
**Pull request by YasunoriMATSUOKA**

### CODE REVIEW

File: .github/workflows/cd-firebase.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-
+      - run: npm ci
+      - run: npm run lint
+      - run: npm run build -- --configuration=production
+      - run: npm run test -- --no-watch --no-progress --browsers=ChromeHeadlessCI
+      - run: npx firebase deploy --token=${{ secrets.FIREBASE_TOKEN }}
```
Comment (next-web-technology-admin) [inline .github/workflows/cd-firebase.yml:32]: Need to add only-hosting option now?

**Review by next-web-technology-admin — CHANGES_REQUESTED** [review]
Need to fix

File: .github/workflows/cd-firebase.yml (hunk 1 above)
Comment (YasunoriMATSUOKA) [reply .github/workflows/cd-firebase.yml]: Done

Comment (next-web-technology-admin) [reply .github/workflows/cd-firebase.yml]: OK

**Review by next-web-technology-admin — APPROVED** [review]
OK

**Pull request merged**